x-pack/plugins/apm/public/components/shared/Links/ClientSideMonitoringLink.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink, EuiLinkAnchorProps } from '@elastic/eui';
+import React from 'react';
+import { useApmPluginContext } from '../../../hooks/useApmPluginContext';
+
+export function ClientSideMonitoringLink(props: EuiLinkAnchorProps) {
+  const { core } = useApmPluginContext();
+  const { basePath } = core.http;
+  const href = basePath.prepend(`/app/csm`);
+
+  return <EuiLink {...props} href={href} />;
```

Comment:
Should this use onClick and navigateToUrl to avoid page reload? https://www.elastic.co/guide/en/kibana/master/kibana-navigation.html#navigating-between-kibana-apps

Reply:
I made this open the CSM app in new tab, which is always a full page load. I'm not totally sure if we should open a new tab, so we could end up going with your suggestion.

Reply:
We're using the `RedirectAppLinks` component so if it doesn't open in a new tab it will not cause a page reload.

Reply:
I'm leaning toward not opening in new tab - but would be good to know if there is a consensus around this in Kibana.

Reply:
I don't think we should open in a new tab.